### DISCUSSION TITLE
Putty Linux Terminal emulator for Windows OS

### DISCUSSION THREAD

**Discussion by ArinzeElvis** [learndevopsonline/learndevopsonline#1555] [Q&A]
Hello @r-devops 

I successfully connected the Putty.exe terminal for the EC2 Instance I created.
However, I have not being able to connect again, even after a couple of cache clearing and system restart, I keep getting "Access Denied"  for a couple of days now. Please note I am sure I have not missed any step in setting up my EC2 Instance or missed the connection credentials. Kindly help assist, I am behind in my practice.
[Error - DevOps.docx](https://github.com/learndevopsonline/learndevopsonline/files/13222021/Error.-.DevOps.docx)

**Comment by ArinzeElvis** [comment]
@r-devops. My mistake, I just realized what I have been missing.  I was not entering the user name right.

**Reply by sreedharm07** [reply]
ok
